src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcAccountMenu.scala.html

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  *@
 
-@this(hmrcNotificationBadge: HmrcNotificationBadge)
+@import uk.gov.hmrc.hmrcfrontend.config.AccountMenuConfig
+
+@this(hmrcNotificationBadge: HmrcNotificationBadge)(implicit accountMenuConfig: AccountMenuConfig)
```

Comment:
@kyle-bowden I think we can remove this implicit now that we changed to a helper